packages/lesmis/src/components/DataGrid/DataGrid.js

```diff
@@ -0,0 +1,148 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useTable, useFilters, useSortBy, usePagination } from 'react-table';
+import { Table, TableHead, TableRow, TableBody, TableSortLabel } from '@material-ui/core';
+import { TableCell, StyledTableRow } from '@tupaia/ui-components';
```

Comment:
What it's importing from where seems a little hard to work out if I were creating another data grid somewhere else. I need most of the Table stuff from mui, but the TableCell (unstyled) and StyledTableRow (styled) from ui-components. 

Personally don't think any of the styling used in LESMIS is very LESMIS-specific, and we could pull the DataGrid up to the ui-components layer?